dotnet/samples/Demos/ContentSafety/README.md

```diff
@@ -0,0 +1,30 @@
+# Azure AI Content Safety and Prompt Shields service example
+
+[Azure AI Content Safety](https://learn.microsoft.com/en-us/azure/ai-services/content-safety/overview) detects harmful user-generated and AI-generated content in applications and services. Azure AI Content Safety includes text and image APIs that allow to detect material that is harmful.
```

Comment:
I would add an introduction to what this Demostration is something like.

This Application Demostrates how to use Semantic Kernel Filters to leverage Content Safety and Shielding combining services like: `X` and `Y`.